planning/behavior_path_planner/autoware_behavior_path_lane_change_module/src/scene.cpp

```diff
@@ -1786,26 +1789,30 @@ PathSafetyStatus NormalLaneChange::isLaneChangePathSafe(
 {
   autoware_utils::ScopedTimeTrack st(__func__, *time_keeper_);
   constexpr auto is_safe = true;
+  constexpr auto is_moving_object_behind_ego = true;
+  if (ego_predicted_paths.empty()) {
+    return {is_safe, !is_moving_object_behind_ego};
+  }
 
-  const auto check_for_colliding_objects = [&](const auto & objects) {
-    return find_colliding_object_if_all_paths_collide(
-      lane_change_path, ego_predicted_paths, objects, rss_params, debug_data, is_approved);
-  };
+  const auto check_for_colliding_objects =
+    [&](decltype(ego_predicted_paths) & ego_pred_paths, const auto & objects) {
+      return find_colliding_object_if_all_paths_collide(
+        lane_change_path, ego_pred_paths, objects, rss_params, debug_data, is_approved);
+    };
 
   const auto check_for_moving_objects = [this](const auto & object) {
     return utils::lane_change::is_moving_object(common_data_ptr_, object);
   };
 
   if (
-    const auto found_colliding_objects_opt =
-      check_for_colliding_objects(collision_check_objects.trailing)) {
+    const auto found_colliding_objects_opt = check_for_colliding_objects(
+      {ego_predicted_paths.front()}, collision_check_objects.trailing)) {
```

Comment:
is it okay to only check on predicted path ? :thinking:

Reply:
Can you further elaborate on this?

Reply:
notes are added to readme https://github.com/autowarefoundation/autoware_universe/pull/11342/commits/3d704e8170e364cdec3ec77b4102573774e9b2d5